lib/pdf/elements/list.rb

```diff
@@ -0,0 +1,18 @@
+class Pdf::Elements::List
+  def self.render(pdf, element, indent: 0)
+    ordered = /ol/ =~ element.name
+
+    index = 0
+    pdf.indent indent do
+      element.css("> li").each do |li|
+        inner_list = li.children.css("ul,ol")
```

Comment:
When you do `#css` won't this search through the whole tree?

```
<ul>
  <li>
    <div>
      <ul>...</u>
```

Wouldn't this match the inner `ul`?

Also I'm not sure this is an issue because I don't think it is possible to do that with markdown. Just curious.